kahuna/public/js/components/gr-image-usage/gr-image-usage.css

```diff
@@ -0,0 +1,35 @@
+.gr-image-usage__field_title {
```

Comment:
Yes. It looks a bit out of place because it hasn't been restricted to `gr-image-usage-list` like rules below. Updating.